app/js/components/manageApps/Addon.jsx

```diff
@@ -337,7 +338,7 @@ class Addon extends Component {
             <h3 id="addon-name">{app.name}</h3>
             <p id="addon-description">{app.description}</p>
             <div className="row">
-              {app.uid ?
```

Comment:
I remember as if there was a commit where one of you explicitly changed this from uuid to uid. Can you cross check from all your team mates?

Reply:
Let me check and get back to you

Reply:
The PR has been updated to a more consistent format for checking the addon type